src/main/java/bio/terra/common/Column.java

```diff
@@ -20,6 +20,13 @@ public Column(Column fromColumn) {
         this.arrayOf = fromColumn.arrayOf;
     }
 
+    public static Column toSnapshotColumn(Column datasetColumn) {
+        return new Column()
+            .name(datasetColumn.getName())
+            .type(datasetColumn.getType())
+            .arrayOf(datasetColumn.isArrayOf());
+    }
```

Comment:
Now that it's here, it no longer needs to be `static`.
```java
    public Column toSnapshotColumn() {
        return new Column()
            .name(name)
            .type(type)
            .arrayOf(arryOf);
    }
```

Reply:
There is an intellij refactor action that will change static to member functions and fix up any call sites. There's also one that will move methods between classes, so both changes can be done using intellij refactoring, which makes changes like this much easier.